src/util/source_location.h

```diff
@@ -130,6 +135,12 @@ class source_locationt:public irept
     set(ID_comment, comment);
   }
 
+  // for switch case number
```

Comment:
Please explain here what this is meant to represent. The value `n` of a `case n:`? An auto-generated id? Any, depending on the caller?